### ISSUE TITLE
Did you apply normalization?

### ISSUE THREAD

**Issue by vietvo89** [locuslab/smoothing#12]
Did you really use normalization when training your model? 

In your `architecture.py`, I found that you used normalize_layer with pre-defined `mean` and `std`. However, when testing your pre-trained models with normalized inputs, the models failed to classify all. However, if removing normalize_layer or no normalization, the models can work correctly.

```
    elif arch == "cifar_resnet110":
        model = resnet_cifar(depth=110, num_classes=10).cuda()
    normalize_layer = get_normalize_layer(dataset)
    return torch.nn.Sequential(normalize_layer, model)
```
